src/containers/Nodes/Nodes.scss

```diff
@@ -17,6 +17,8 @@
     }
 
     &__groups-wrapper {
-        padding-right: 20px;
+        --nodes-side-padding: var(--g-spacing-5);
+        width: calc(100% + var(--nodes-side-padding));
+        padding-right: var(--nodes-side-padding);
```

Comment:
Why don't you just remove padding here?

Such approach (100%+ width, hidden paddings, negative margins, transforms) is cursed, it causes unexpected behavior and bugs long-term - it looks like you fix cluster controls and autorefresh layout bugs every 2-3 weeks

Reply:
Really, thats time to get rid of this evil.
So, I've done it. Stand redeployed.

Reply:
You didn't delete groups wrapper classes